integration-libs/opf/b2b-checkout/components/opf-b2b-checkout-payment-and-review/opf-b2b-checkout-payment-and-review.component.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { Cart } from '@spartacus/cart/base/root';
+import { Observable, take, map, filter, combineLatest } from 'rxjs';
+import { Card } from '@spartacus/storefront';
+import { CheckoutStepType } from '@spartacus/checkout/base/root';
+import { OpfCheckoutPaymentAndReviewComponent } from '@spartacus/opf/checkout/components';
+
+@Component({
+  selector: 'cx-opf-b2b-checkout-payment-and-review',
+  templateUrl: './opf-b2b-checkout-payment-and-review.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: false,
+})
+export class OpfB2bCheckoutPaymentAndReviewComponent
+  extends OpfCheckoutPaymentAndReviewComponent
+  implements OnInit
+{
+  checkoutStepTypePaymentType = CheckoutStepType.PAYMENT_TYPE;
+
+  paymentType$ = this.activeCartFacade
+    .getActive()
+    .pipe(map((cart: Cart) => cart.paymentType));
+
+  get poNumber$(): Observable<string | undefined> {
```

Comment:
Getter or assigned property?

Reply:
Switched `poNumber$` to assigned property.